packages/message-encryption/src/ecies.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { IProtoMessage } from "@waku/interfaces";
+import { proto_message } from "@waku/proto";
```

Comment:
Good reason to re-export `proto_message.WakuMessage` from `@waku/interfaces` as `IProtoMessage`.